README.md

```diff
@@ -42,6 +42,42 @@ job("foo") {
 }
 ```
 
+## Declarative pipeline
+
+Examples:
```

Comment:
Would you mind adding a sentence or two introducing this example? I think this would make the documentation  a bit more readable.

Reply:
That's a good idea. I have fixed that now.